.ci/idea-inspection.sh

```diff
@@ -49,10 +55,12 @@ echo "Checking results ..."
 PROBLEM_COUNT=$(grep -R "<problems" "$RESULTS_DIR"/ | cat | wc -l )
 
 if [[ $PROBLEM_COUNT -gt 0 ]] && [[ "$CIRCLECI" == "true" ]]; then
-    echo -e "\n\n\n\n"
+    echo ""
     echo "ATTENTION GITHUB CONTRIBUTOR:"
     echo "There are inspection problems."
-    echo "Review results in 'ARTIFACTS' tab in the CircleCI UI above."
+    echo "Review results in 'ARTIFACTS' tab in the CircleCI UI above, or follow the link below:"
+    CIRCLECI_APP_URL="https://app.circleci.com/pipelines/github/checkstyle/checkstyle"
+    echo "$CIRCLECI_APP_URL/17861/workflows/$CIRCLE_WORKFLOW_ID/jobs/$CIRCLE_BUILD_NUM/artifacts"
```

Comment:
>17861

What is this? Can it be different?

Reply:
I tried different numbers in a few different PRs (my own, and others) and it still worked. It seems like this number doesn’t actually matter. Example: https://app.circleci.com/pipelines/github/checkstyle/checkstyle/1/workflows/e75ebb44-1f20-4e32-84a9-3fa3215872a9/jobs/265866/artifacts

https://app.circleci.com/pipelines/github/checkstyle/checkstyle/17/workflows/e75ebb44-1f20-4e32-84a9-3fa3215872a9/jobs/265866/artifacts

Reply:
if does not matter, let make it `1` as clear sign that for now CirlceCi ignore this

Reply:
Done, left a comment as well.